scripts/e2e-test-regions.json

```diff
@@ -0,0 +1,21 @@
+[
+  "ap-northeast-2",
```

Comment:
'ap-northeast-3' is missing.

Reply:
That is not an amplify supported region.

Reply:
Looks, that's right. But I don't understand why CLI use this region with their E2E tests.
https://github.com/aws-amplify/amplify-cli/blob/d3261769506403c3a3222a5af2eb213b0f3b7844/packages/amplify-e2e-core/src/configure/index.ts#L26-L46